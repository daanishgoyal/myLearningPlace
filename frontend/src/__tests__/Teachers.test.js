import { render } from "@testing-library/react";
import Teachers from "../components/teachers/Teachers.component";

it("teacher page comes or not", () => {
    const { container } = render(<Teachers />);
    expect(container).toMatchSnapshot();
});
