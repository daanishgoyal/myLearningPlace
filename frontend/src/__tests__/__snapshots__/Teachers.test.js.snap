// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="col"
    />
    <div
      class="col"
    >
      <div
        class="row-sm-1 sticky"
      >
        <div
          class="col-sm-12"
        >
          <form
            class="row gy-2 gx-3 align-items-center bg-success p-5 m-5 text-white"
          >
            <div
              class="row justify-content-md-center"
            >
              <div
                class="row"
              >
                <div
                  class="col"
                />
                <div
                  class="col-sm-5"
                >
                  <h3>
                    Search for online/home tutors
                  </h3>
                </div>
                <div
                  class="col"
                />
              </div>
              <div
                class="row"
              >
                <div
                  class="col-sm-2"
                />
                <div
                  class="col-sm-3 m-3"
                >
                  <select
                    class="form-control"
                    id="skill"
                    name="skill"
                  >
                    <option
                      class="text-secondary"
                      value=""
                    >
                      Subject / Skill
                    </option>
                  </select>
                </div>
                <div
                  class="col-sm-3 m-3"
                >
                  <div
                    class="input-group"
                  >
                    <select
                      class="form-control"
                      id="city"
                      name="city"
                    >
                      <option
                        class="text-secondary"
                        value=""
                      >
                        Location
                      </option>
                    </select>
                  </div>
                </div>
                <div
                  class="col-sm-2 m-3"
                >
                  <button
                    class="btn btn-primary"
                    disabled=""
                    type="submit"
                  >
                    Search
                  </button>
                </div>
              </div>
            </div>
          </form>
        </div>
      </div>
      <div
        class="row-sm-12"
      >
        <div
          class="col-sm-12"
        >
          <div
            class="row gy-2 gx-3 align-items-center bg-dark p-5 m-5 text-white"
          />
        </div>
      </div>
      <div
        class="row-sm-10"
      >
        <div
          class="col-sm-12"
        >
          <div
            class="row gy-2 gx-3 align-items-center p-5 m-5 text-white"
          />
        </div>
      </div>
    </div>
    <div
      class="col"
    />
  </div>
</div>
`;
